caniusepython3/dependencies.py

```diff
@@ -24,6 +24,8 @@
 import logging
 
 
+INDEX_URL = "https://pypi.org/pypi"
```

Comment:
This is defined twice: here and in `caniusepython3.pypi`. Since this module already imports `.pypi`, please define the constant there and use the constant here.